src/Process/StandardSymfonyProcessFactory.php

```diff
@@ -26,12 +27,10 @@ public function startProcess(
         ?array $environmentVariables,
         callable $processOutput
     ): Process {
-        $process = new Process(
+        $process = new PhpSubprocess(
```

Comment:
Currently `$command` contains the PHP executable, so for `PhpSubprocess` to work we would need to:

- remove the PHP executable from `$command`
- pass it as a parameter to `PhpSubprocess`